tests/check_doctest_names.py

```diff
@@ -14,8 +14,12 @@
     >>> import vtk
     >>> import pyvista
     >>> offset = np.array([0, 9])
-    >>> cells = np.array([8, 0, 1, 2, 3, 4, 5, 6, 7, 8, 8, 9, 10, 11, 12, 13, 14, 15])
-    >>> cell_type = np.array([vtk.VTK_HEXAHEDRON, vtk.VTK_HEXAHEDRON], np.int8)
+    >>> cells = np.array(
+    ...     [8, 0, 1, 2, 3, 4, 5, 6, 7, 8, 8, 9, 10, 11, 12, 13, 14, 15]
```

Comment:
Same comment as previous, now that we are breaking this into multiple lines, the `cells` array could be shown in a human interpretable form with each cell on it's own line.

Reply:
Fixed in 062fe1ab41418178050ae4121a75a2442a22f5f7.